### ISSUE TITLE
Dependencies in CI

### ISSUE THREAD

**Issue by nosnickid** [GSS-Cogs/dd-cms#109]
We've got a `Dockerfile-test` that uses `COPY package.json mrs-developer.json (etc)` and `RUN yarn install` to try and create a cached `node_modules` dir.

Jenkins mounts over that with the workspace dir, so we don't get the docker layer cache benefits we're expecting. 

Each branch could have different dependencies and addons.

Seems like a general problem with CI, and with Jenkins in particular. The mrs-developer and workspaces setup in volto make it a bit fiddlier still for us.

We should probably at least remove the duplicate effort of running `yarn install` in the container build.

**Comment by nosnickid** [comment]
Alex suggested export NODE_MODULES=/app/node_modules might enable the Docker layer caching to work as we want.

**Comment by nosnickid** [comment]
https://nodejs.org/api/modules.html#all-together has the algo that node uses to  resolve files. It mentions a NODE_PATH env var, but also says it's not really useful these days.

I tried it anyway (`NODE_PATH=/app/node_modules yarn test`) and it didn't work.

I've had a good dig around docs for something to let us override this path and haven't found anything that looks useful.

So I'm going to just use a softlink in the container. That seems to work.  

It had one issue with our MultilingualRedirector override that was trying to import locale files. But, we didn't actually customize that component, so I've removed our override for it. And it was doing something fairly unique - a  `import("~/locale/")` call - so it's not likely to be a recurring problem.
